src/pages/claim-details/claim-actions.tsx

```diff
@@ -52,7 +52,11 @@ export default function ClaimActions(props: Props) {
 
   const handleAppeal = async () => {
     setStatus('submitting');
-    const tx = await handleTransactionError(arbitratorProxy.appealKlerosArbitratorRuling(claim.claimId));
+    // TODO DAO-176 Handle in appeal confirmation modal
+    const appealCost = parseEther('0.004');
```

Comment:
Mhh, what is this cost? Is it some fixed constant for appeals?

Reply:
This is just what the current mock arbitrator expects, which will change when Burak gets to implementing the proper KlerosLiquid mock arbitrator. At that point I can implement DA0-176 which will handle the appeal cost properly.

Reply:
I see. I missed the context for that TODO.